src/Components/Base/Icon/index.tsx

```diff
@@ -4,14 +4,18 @@ import IconProps from './type';
 
 const Icon = forwardRef(
   (
-    { name, isFill = true, ...props }: IconProps,
+    { name, isFill = true, className = 'hi', ...props }: IconProps,
```

Comment:
```suggestion
    { name, isFill = true, className, ...props }: IconProps,
```
기본 클래스명은 빈 문자열이 좋을 꺼 같습니다!!

Reply:
수정했습니다~!